Fixes platform services (#1611)
SoftwareAG/c8y-docs: content/change-logs/platform-services/2023-12-06_log-files-stored-in-platform-can-now-be-downloaded.md

---
date: 2023-12-06T16:00:12.648Z

> title: Log files stored in platform can now be downloaded

Reviewer: ```suggestion
title: Log files of devices can now be downloaded
```
